iscdhcp/files/dhcpd.hosts

```diff
@@ -12,7 +12,9 @@
 host {{ h.name }} {
   hardware ethernet {{ h.mac }};
   fixed-address {{ h.ipaddr }};
+  {%- if h.get('insert-ddns-hostname', True) %} 
```

Comment:
sorry, you're right. I haven't reviewed this change carefully. ``ddns-hostname`` is already set in the line below. :)